modules/ROOT/partials/gcs-credentials.adoc

```diff
@@ -0,0 +1,28 @@
+. Ensure you have a Google account and a project created in the Google Cloud Platform (GCP).
+.. Install the `gcloud` CLI by following the instructions in the Google official documentation -- link:https://cloud.google.com/sdk/docs/install[Install the gcloud CLI].
+.. Create a service account and a service account key using Google official documentation -- link:https://cloud.google.com/iam/docs/service-accounts-create[Create service accounts] and link:https://cloud.google.com/iam/docs/creating-managing-service-account-keys[Creating and managing service account keys].
+.. Download the JSON key file for the service account.
+.. Set the `GOOGLE_APPLICATION_CREDENTIALS` and `GOOGLE_CLOUD_PROJECT` environment variables to the path of the JSON key file and the project ID, respectively:
++
+[source,shell]
+----
+export GOOGLE_APPLICATION_CREDENTIALS="/path/to/keyfile.json"
+export GOOGLE_CLOUD_PROJECT=YOUR_PROJECT_ID
+----
+.. Authenticate the `gcloud` CLI with the service account by running the following command:
++
+[source,shell]
+----
+gcloud auth activate-service-account SERVICE_ACCOUNT@DOMAIN.COM --key-file=$GOOGLE_APPLICATION_CREDENTIALS --project=$GOOGLE_CLOUD_PROJECT
```

Comment:
maybe mention that `SERVICE_ACCOUNT@DOMAIN.COM` should be the one from a couple of steps ago